cleanlab_studio/studio/studio.py

```diff
@@ -396,6 +397,93 @@ def download_embeddings(
         """
         return np.asarray(api.download_array(self._api_key, cleanset_id, "embeddings"))
 
+    def create_enrichment_project(
+        self,
+        name: str,
+        dataset_id: str,
+    ) -> enrichment.EnrichmentProject:
+        """
+        Creates a Cleanlab Studio Enrichment Project.
+
+        Args:
+            name (str): Name of the enrichment project to create.
+            dataset_id (str): ID of dataset to be enriched.
+
+        Returns:
+            EnrichmentProject: The [EnrichmentProject](../enrichment#class-enrichmentproject) object
+            for the new enrichment project.
+        """
+        enrichment_project_dict = api.create_enrichment_project(
+            api_key=self._api_key,
+            name=name,
+            dataset_id=dataset_id,
+        )
+
+        return enrichment.EnrichmentProject(
+            api_key=self._api_key,
+            id=enrichment_project_dict["id"],
+            name=enrichment_project_dict["name"],
+            created_at=enrichment_project_dict["created_at"],
+        )
+
+    def delete_enrichment_project(self, project_id: str) -> None:
+        """
+        Deletes an Enrichment Project from Cleanlab Studio.
+
+        Args:
+            project_id: ID of enrichment project to delete.
+        """
+        api.delete_enrichment_project(self._api_key, project_id=project_id)
+        print(f"Successfully deleted enrichment project: {project_id}")
+
+    def get_enrichment_project(
+        self,
+        project_id: str,
+    ) -> enrichment.EnrichmentProject:
+        """
+        Get an EnrichmentProject objection for a given Cleanlab Studio Enrichment Project's ID.
+
+        Args:
+            project_id (str): ID of the enrichment project.
+
+        Returns:
+            EnrichmentProject: The [EnrichmentProject](../enrichment#class-enrichmentproject) object
+            for the enrichment project.
+        """
+        enrichment_project_dict = api.get_enrichment_project(
+            api_key=self._api_key,
+            project_id=project_id,
+        )
+
+        return enrichment.EnrichmentProject(
+            self._api_key,
+            id=project_id,
+            name=enrichment_project_dict["name"],
+            created_at=enrichment_project_dict["created_at"],
+        )
+
+    def get_enrichment_projects(
+        self,
+    ) -> List[enrichment.EnrichmentProject]:
+        """
+        Get a list of all EnrichmentProjects.
+
+        Returns:
+            List[EnrichmentProject]: A list of [EnrichmentProject](../enrichment#class-enrichmentproject) objects.
+        """
+        enrichment_project_dicts = api.list_all_enrichment_projects(
+            api_key=self._api_key,
+        )
+
+        return [
+            enrichment.EnrichmentProject(
+                self._api_key,
+                id=enrichment_project_dict["id"],
+                name=enrichment_project_dict["name"],
+            )
```

Comment:
i think add timestamp is better. user can sort in their own way and find things they need. 
suggest + `created_at` and `updated_at`. to make it consistent with other methods return.

Reply:
I've added `created_at`.